docs/examples/tutorial/clibraries/queue3.pyx

```diff
@@ -38,11 +38,9 @@ cdef class Queue:
             self.append(value)
 
     cdef extend_ints(self, int* values, size_t count):
-        cdef size_t i
-        for i in range(count):
-            if not cqueue.queue_push_tail(
-                    self._c_queue, <void*> values[i]):
-                raise MemoryError()
+        cdef int value
+        for value in values[:count]:  # It is possible to slice pointers in Cython.

```

Comment:
The change looks good, but the comment doesn't feel very useful. "It is possible" ? Ah, good to know – but why would I do it, and what does it do?

I think this would be a better comment: "Slicing pointer to limit the iteration boundaries."